.azure-pipelines/ci-build.yml

```diff
@@ -243,6 +243,13 @@ extends:
       dependsOn: build
       jobs:
         - deployment: deploy_nuget
+          templateContext:
+            type: releaseJob
+            isProduction: true
+            inputs:
+            - input: pipelineArtifact
```

Comment:
With the pipeline input here, this means we can probably drop the `DownloadPipelineArtifact@2` step in line 268

Reply:
I wasn't sure it'd do the work for us, thanks for confirming